Platform.XamarinAndroid/Tests.XamarinAndroid/Tests.XamarinAndroid.csproj

```diff
@@ -78,47 +78,29 @@
     <AndroidResource Include="Resources\drawable-xxxhdpi\Icon.png" />
   </ItemGroup>
   <Import Project="..\..\Shared\Tests.Shared\Tests.Shared.projitems" Label="Shared" Condition="Exists('..\..\Shared\Tests.Shared\Tests.Shared.projitems')" />
-  <Import Project="..\..\Shared\Tests.Sync.Shared\Tests.Sync.Shared.projitems" Label="Shared" />
+  <Import Project="..\..\Shared\Tests.Sync.Shared\Tests.Sync.Shared.projitems" Label="Shared" Condition="'$(RealmNoSync)' != 'true'" />
```

Comment:
It's very minor but I hate how we have double negative here: `xxxNoxxx is not true`. Don't insist on changing it though - it's unlikely anyone will be looking at those files anyway :)